flybywiresim/installer
# feat: add possibility of getting latest track versions from fragmenter distribution manifest

## Summary of Changes


 - Add possibility of getting latest track versions from fragmenter distribution manifest
 - Clean up install state detection logic
 - Clean up update checking logic
 - Add developer setting for forcing use of local configuration
 
## Testing Instructions
- Make sure addon installing, updating, version switching, removal, etc. works correctly
- Make sure you cannot get into a broken state by cancelling the install/update/version switch
- Make sure those operations are correctly blocked when SImBridge or the sim are running
